.circleci/scripts/should_run_job.py

```diff
@@ -28,6 +28,8 @@
     # Caffe2 ONNX
     'caffe2-onnx-py2-gcc5-ubuntu16.04',
     'caffe2-onnx-py3.6-clang7-ubuntu16.04',
+    'caffe2-onnx-py3.6-part1-clang7-ubuntu16.04',
```

Comment:
can we use name 

caffe2-onnx-py3.6-main-test-clang7-ubuntu16.04 and caffe2-onnx-py3.6-ort-test-clang7-ubuntu16.04

instead of part1 and part2

Reply:
I'm actually dividing ORT tests by opset version. This shows to be the most efficient distribution since ORT (torchvision) tests take many times longer than the other set of tests, and we run those for 6 opset versions (and we're going to add more going forward).